cms/templates/certificates.html

```diff
@@ -67,10 +67,13 @@ <h2 class="sr title">${_("Certificates")}</h2>
-            <p>${_("Click {em_start}Add your first certificate{em_end} to add a certificate configuration. Upload the organization logo to be used on the certificate, and specify at least one signatory. You can include up to four signatories for a certificate. You can also upload a signature image file for each signatory. {em_start}Note:{em_end} Signature images are used only for verified certificates. Optionally, specify a different course title to use on your course certificate. You might want to use a different title if, for example, the official course name is too long to display well on a certificate.") .format(em_start='<strong>', em_end="</strong>")}</p>
-            <p>${_("Select a course mode and click {em_start}Preview Certificate{em_end} to preview the certificate that a learner in the selected enrollment track would receive. When the certificate is ready for issuing, click {em_start}Activate.{em_end} To stop issuing an active certificate, click {em_start}Deactivate{em_end}.").format(em_start='<strong>', em_end="</strong>")}</p>
-            <p>${_(" To edit the certificate configuration, hover over the top right corner of the form and click {em_start}Edit{em_end}. To delete a certificate, hover over the top right corner of the form and click the delete icon. In general, do not delete certificates after a course has started, because some certificates might already have been issued to learners.").format(em_start="<strong>", em_end="</strong>")}</p>
+            <p>${_("Use this page to create a certificate for each course mode.")}</p>
+            <p>${_("Select {em_start}Add your first certificate{em_end} to create a new certificate. You can then specify a course title to use if the course's official title is too long to be displayed well on a certificate.").format(em_start='<strong>', em_end="</strong>")}</p>
+            <p>${_("For verified certificates, you must specify between one and four signatories.").format(em_start='<strong>', em_end="</strong>")}</p>
+            <p>${_("To edit or delete a certificate, hover over the top right corner of the form and select {em_start}Edit{em_end} or the delete icon.").format(em_start="<strong>", em_end="</strong>")}</p>
+            <p>${_("{em_start}Do not{em_end} delete certificates after a course has started; learners who have already earned certificates will no longer be able to access them.").format(em_start="<strong>", em_end="</strong>")}</p>
+            <p>${_("To test how a certificate will appear, select a course mode and select {em_start}Preview Certificate{em_end}.").format(em_start='<strong>', em_end="</strong>")}</p>
+            <p>${_("A course administrator selects {em_start}Activate{em_end}  to begin issuing certficates. Course teams cannot edit an activated certificate.").format(em_start='<strong>', em_end="</strong>")}</p>
```

Comment:
"To begin issuing certificates for this course, a course team member with the Admin role selects Activate. Course teams cannot edit an activated certificate."